index.html

```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
+
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-structure">
+          <img src="images/yoga.png" class="img-displayer" alt="first_photo">
+
+          <h1 class="project-title">Tonic</h1>
+
+          <ul class="tags">
+            <li class="client">CANOPY</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">Back End Dev</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">2015</li>
+          </ul>
+
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-structure">
+          <img src="images/ball.png" class="img-displayer" alt="first_photo">
+
+          <h1 class="project-title">Multi-Post
+            Stories</h1>
+
+          <ul class="tags">
+            <li class="client">CANOPY</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">Back End Dev</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">2015</li>
+          </ul>
+
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1 class="about-myself">About <br>Myself</h1>
+
+      <p class="myself">
+        Hello I’m a software developer!
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience!
+        If you like what you see and have a
+        project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h3 class="met">LET'S CONNECT</h3>
+
```

Comment:
kindly add some space between the paragraph and the h3 tag 
![wrong2](https://user-images.githubusercontent.com/58376020/121585835-51d52c80-ca33-11eb-8ffa-cef82f10bd30.png)

Reply:
![image](https://user-images.githubusercontent.com/78529917/121605489-25320c80-ca55-11eb-815c-071ca2f497e4.png)
what's wrong in here